src/lib/components/Popover.svelte

```diff
@@ -99,6 +104,10 @@
     border-radius: var(--border-radius);
 
     box-shadow: var(--overlay-box-shadow);
+
+    &.with-border {
+      border: var(--tooltip-border-size) solid var(--tooltip-border-color);
```

Comment:
the border color in the screenshot of the PR visually don't exactly match the outcome of Figma, the contrast isn't as visually distinctive. is that expected?

Reply:
The border color is exactly what it is in Figma, but the background color is not.
I was going to change the background color in another PR, but I've done it in this PR now.
Now it looks like this:
<img width="284" alt="image" src="https://github.com/dfinity/gix-components/assets/122978264/3bb5aa50-94d5-403d-b4b9-58564cc27eda">

Reply:
Contrast is better I think, thanks!